<?xml version="1.0"?>
<launch>
  <arg name="_name" default="adrianohrl"/>
  <group ns="$(arg _name)">
    <node name="ruler_test_node" pkg="ruler_test" type="ruler_test_node" output="screen">
      <rosparam file="$(find ruler_test)/config/ruler_test_params.yaml" command="load"/>
    </node>
  </group>
</launch>
